src/kirby/components/badge/badge.component.tns.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../scss/native-imports";
+
+Label {
```

Comment:
Change to styling the host instead or else it will not work with icons inside it.
Since {N} doesn't support the `:host` selector yet, we can do it the same way we do it for button:
https://github.com/kirbydesign/designsystem/blob/master/src/kirby/components/button/button.component.tns.global.scss
https://github.com/kirbydesign/designsystem/blob/master/src/kirby/scss/native/_native-common.scss